meilisearch-core/src/query_builder.rs

```diff
@@ -109,12 +120,30 @@ impl<'c, 'f, 'd, 'fa, 'i> QueryBuilder<'c, 'f, 'd, 'fa, 'i> {
             None => None
         };
 
+        let facet_count_docids = match self.facets {
```

Comment:
Can you add a comment just to explain a little bit what this block of code do?
It seems like it retireves the documents ids associated with facets that the user requested.